packages/alfa-rules/src/sia-r83/rule.ts

```diff
@@ -93,32 +92,41 @@ function isPossiblyClipped(device: Device): Predicate<Element> {
         .some((whiteSpace) => whiteSpace.value === "nowrap")
     ) {
       return style
-        .computed("overflow-x")
-        .some(
-          (overflow) => overflow.value === "hidden" || overflow.value === "clip"
+        .cascaded("overflow-x")
```

Comment:
We should explain that why we need cascaded value rather than computed, since it is a bit unusual.

Reply:
`overflow-x` doesn't inherit and computes as specified so this is just a micro optimisation. That's also worth pointing out though.

Reply:
I forgot that computed values are pulled directly from cache just as cascaded values are pulled directly from the property map so we might as well grab the computed value and save two lines of code 😄